src/App.js

```diff
@@ -325,8 +326,17 @@ export class ResourceEditor extends React.Component {
     this.setState({ currentStep: newStep });
   };
 
-  handleUpload = () => {
-    alert("Uploaded Successfully");
+  handleUpload = async () => {
+    axios({
+      method: 'post',
+      url: `http://localhost:3001/api/dataset/${this.state.datasetId}`,
```

Comment:
Is it advisable to use localhost here, what happens when we embed in portal app?

Reply:
I forgot to remove the entire url, that was a test...